### DISCUSSION TITLE
Correction affichage nom commune sur page équipment

### DISCUSSION THREAD

**Discussion by elmerdu69** [OUARZA/Propluvia#48] [Show and tell]
Hello,
J'ai corrigé la façon d'afficher le nom de la commune sur la page équipement et ça devrait bien fonctionner maintenant.
J'en ai profité pour faire afficher le nom du département en plus.
ça donne ça au final :
![image](https://github.com/OUARZA/Propluvia/assets/115868585/5d9117c0-59e3-4589-8292-c30bed3302c1)
J'ai fait simple en fonction de ce que j'ai trouvé. Si tu n'est pas fan des pavés bleu, dis le moi ou bien modifie le de ton coté.

**Comment by OUARZA** [comment]
Hello,
Je préfère le format Tile.
J'ai voulu l'intégrer, puis plus rien...
J'ai remis ton code, mais rien ne s'affichage...
Je sais pas pourquoi.

**Reply by elmerdu69** [reply]
salut, partage le bout de code que tu as essayé stp

**Reply by elmerdu69** [reply]
tiens donc je viens de faire la mise à jour que tu as poussé, et plus rien de s'affiche non plus de mon coté...
je restaure la version précédente pour voir ce qu'il se passe

**Comment by OUARZA** [comment]
Je suis désolé, j'ai juste modifié 
`<span class="eqLogicAttr label label-info" style="font-size:1em;cursor: default;" data-l1key="configuration" data-l2key="departement"></span>`
en
`<span class="col-sm-3 control-label" style="font-size:1em;text-align: left;" data-l1key="configuration" data-l2key="departement"></span>`

**Reply by elmerdu69** [reply]
je viens de voir, pas de soucis.
si je comprend bien, il faut que tu gardes `class="eqLogicAttr`
mon code d'origine est `<span class="eqLogicAttr label label-info" style="font-size:1em;cursor: default;" data-l1key="configuration" data-l2key="departement">`
je vais essayer de modifier le format de mon coté, mais tu peux essayer aussi du tiens

**Reply by elmerdu69** [reply]
est ce que comme ça l'affichage du nom de la commune, ça te va ? 
![image](https://github.com/OUARZA/Propluvia/assets/115868585/a9785977-4db6-4831-b9f1-86c3b4865c77)

**Comment by OUARZA** [comment]
Oui !
Merci beaucoup

**Reply by elmerdu69** [reply]
parfait alors !
je fais la modif complète et la pousse dans quelques minutes

**Comment by OUARZA** [comment]
Je comprends pas j'ai modifié dans ma console de dev...

**Reply by elmerdu69** [reply]
je ne sais pas non plus
tu peux valider le PR maintenant
